python/cuml/dask/common/base.py

```diff
@@ -40,12 +40,16 @@
 
 class BaseEstimator(object, metaclass=BaseMetaClass):
 
-    def __init__(self, *, client=None, verbose=False, **kwargs):
+    def __init__(self, *, client=None, **kwargs):
```

Comment:
Sure, it's probably better to keep it as an explicit argument in the `BaseEstimator` class. We just have to make sure the verbosity is systematically transferred from here, otherwise, if done from estimators implementations it might be forgotten.